test/consistent-package-names.ts

```diff
@@ -0,0 +1,23 @@
+import {readdirSync, readFileSync} from 'fs';
+import * as path from 'path';
+
+describe('package name', () => {
+  it('matches the path name', () => {
+    const packagesPath = path.resolve(__dirname, '..', 'packages');
+    const packageNames = readdirSync(packagesPath);
+
+    for (const packageName of packageNames) {
```

Comment:
this will need to change below too. We need to account for the case where the folder is left over from a different branch, locally.

Reply:
see the change here: https://github.com/Shopify/quilt/pull/127/files#diff-08f9f004f75cebe0e8288463aea4b843R19